kura/examples/org.eclipse.kura.example.ble.tisensortag/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,7 @@ Bundle-Version: 1.0.200.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Service-Component: OSGI-INF/*.xml
 Import-Package: org.eclipse.kura;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth;version="[1.1,2.0)",
+ org.eclipse.kura.bluetooth.le;version="1.0.0",
```

Comment:
The previous BLE example was released with version 1.0.100. Since I did some modifications on this bundle I increased the version to 1.0.200.